### PR TITLE
Add ipywidgets_bokeh as a package

### PR DESCRIPTION
**Pull request by hoxbro**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hoxbro** [comment]
@conda-forge-admin, please rerender

**Comment by raybellwaves** [comment]
Haven't looked at this in a while. Want to add yourself (@hoxbro) to https://github.com/hoxbro/ipywidgets-bokeh-feedstock/blob/add_underscore/recipe/meta.yaml#L58 ?

**Comment by hoxbro** [comment]
Thank you. I added myself as a maintainer.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,27 +10,39 @@ source:
...
-    - pip
-    - python >=3.9
-  run:
-    - python >=3.9
-    - bokeh ==3.*
-    - ipykernel ==6.*,!=6.18.0
-    - ipywidgets ==8.*
+outputs:
+  - name: ipywidgets_bokeh
+    script: {{ PYTHON }} -m pip install . -vv
```
Comment (hoxbro) [inline recipe/meta.yaml]: I get a `PYTHON is undefined` error [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=935804&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=508), and I have no idea why.

@jakirkham, is it possible for you to help me with this?

**Comment by raybellwaves** [comment]
Would you mind commenting on this PR what you are trying to do?

Does ipywidgets_bokeh build ipywidgets_bokeh and ipywidgets-bokeh packages for pypi?

Probably not the best option (as you have to manage both) but I wonder if it's worth creating a new ipywidgets_bokeh-feedstock. May help you debug.

I think I'm guessing you are having this feedstock build ipywidgets_bokeh and ipywidgets-bokeh output for conda-forge. Curious why ipywidgets_bokeh is needed?

**Comment by hoxbro** [comment]
> Curious why ipywidgets_bokeh is needed?

This is the name of the package, https://github.com/bokeh/ipywidgets_bokeh/blob/a62bad95be6c3a0ba9bcdf55fb05cce381f07533/setup.py#L41.

The PyPi name does not make any difference between the `-`/`_` like conda does. 

Bokeh uses underscore for their other packages, see https://github.com/conda-forge/jupyter_bokeh-feedstock and their own channel https://anaconda.org/bokeh/ipywidgets_bokeh. 

> Does ipywidgets_bokeh build ipywidgets_bokeh and ipywidgets-bokeh packages for pypi?

I'm trying to change the name to `ipywidgets_bokeh` without removing `ipywidgets-bokeh`. And making `ipywidgets-bokeh` a meta package that pulls in the "true" package `ipywidgets_bokeh`.

File: recipe/meta.yaml (hunk 1 above)
Comment (hoxbro) [reply recipe/meta.yaml]: I found a way by taking inspiration from https://github.com/conda-forge/typing_extensions-feedstock/blob/main/recipe/meta.yaml

**Comment by raybellwaves** [comment]
LGTM. Thanks

**Pull request merged**